pkg/cmd/prune-backups/prune.go

```diff
@@ -36,7 +36,9 @@ type PruneOpts struct {
 }
 
 func NewPruneCommand() *cobra.Command {
-	opts := PruneOpts{RetentionType: "None"}
+	opts := PruneOpts{
```

Comment:
unrelated to the purpose of this PR

Reply:
sure 👍🏽

Reply:
this was already here, it is just the formatting